.murdock

```diff
@@ -76,21 +76,34 @@ get_supported_boards() {
     done | $(_greplist $BOARDS)
 }
 
-# given an app dir as parameter, print "$appdir board" for each supported
-# board. Only print for boards in $BOARDS.
-get_app_board_pairs() {
+get_supported_toolchains() {
+    local board=$1
+
+    if [ "$board" = "native" ]; then
+        echo "gnu llvm"
+    else
+        echo "gnu"
+    fi
```

Comment:
Maybe this construct could be replaced with some `BOARD=$board make info-toolchains-supported` target? This way also the ARM platforms will be build with LLVM.